### ISSUE TITLE
Home: Incorrectly Aligned “See More Collections” Link

### ISSUE THREAD

**Issue by jscholes** [benetech/ImageShare#56]
The “See More Collections” link is right-aligned, but should be left-aligned so that it appears directly below the “Chemistry” image link.

![Screenshot of the “See More Collections” link, right-aligned, directly below the five Explore Collections image links.](https://user-images.githubusercontent.com/6454755/80001160-ac07dc80-8483-11ea-9ea4-4e8d2538a92a.png)

**Comment by clapierre** [comment]
That is fine we can left align that (totally get that for a Low Vision user that is zoomed in may miss that on the right side.

**Comment by sinabahram** [comment]
I don't believe this applies anymore.
